### DISCUSSION TITLE
AMD vs Nvidia

### DISCUSSION THREAD

**Discussion by Nibowin** [alicevision/Meshroom#2571] [Q&A]
Does anyone have examples of the differences between running similarly spec'd NVIDIA and AMD gpu's? I currently have an AMD card and would rather not invest to swap to NVIDIA but if the results are much much worse then I would. Thank you.

**Comment by natowi** [comment]
With ZLUDA it is possible to run Meshroom on AMD: https://github.com/vosen/ZLUDA/issues/79
"ZLUDA is a drop-in replacement for CUDA on non-NVIDIA GPU. ZLUDA allows to run unmodified CUDA applications using non-NVIDIA GPUs with near-native performance."
Here is a build I prepared [Meshroom-2023.3.0-ZLUDA-2024-03.zip](https://github.com/natowi/meshroom_external_plugins/releases/download/test/Meshroom-2023.3.0-ZLUDA-2024-03.zip)
I´d love to get some feedback, as I don´t own a AMD GPU.

**Reply by CarCrasher13** [reply]
what version of Zluda did you use? I am trying to run the new version of meshroom on linux, but it seems that Zluda went thru a remake, and no longer supports meshroom.

**Reply by natowi** [reply]
This https://github.com/vosen/ZLUDA/issues/79#issuecomment-2028079955
Probably https://github.com/vosen/ZLUDA/releases/tag/v3 based on the time
